lib/piggly/compiler/trace_compiler.rb

```diff
@@ -30,6 +31,17 @@ def compile(procedure)
           code = traverse(tree, procedure.oid, tags)
 
           cache.replace(:tree => tree, :code => code, :tags => tags)
+          rescue RuntimeError => e
+            $stdout.puts <<-EXMSG
+            ****
+            Error compiling procedure #{procedure.name}
+            Source: #{procedure.source_path(@config)}
+            Exception Message:
+            #{e.message}
+            ****
+            EXMSG
+          end
+
```

Comment:
Added error message for a failed procedure parse; It just dumps the treetops error which can be a bit terse but it still points you in the right place.